stack build fails with No instance for (MonadFail UpdateParser)

```
No instance for (MonadFail UpdateParser)/home/vyorkin/projects/github/haskell/telegram-bot-simple/src/Telegram/Bot/Simple/UpdateParser.hs:43:10: error:
    • No instance for (MonadFail UpdateParser)
        arising from a use of ‘fail’
    • In the expression: fail "command"
      In a stmt of a 'do' block:
        if "/" `Text.isPrefixOf` t then fail "command" else pure t
      In the expression:
        do t <- text
           if "/" `Text.isPrefixOf` t then fail "command" else pure t
   |
43 |     then fail "command"
   |          ^^^^^^^^^^^^^^
```

Maybe this is because the instance was accidentally deleted https://github.com/fizruk/telegram-bot-simple/commit/cbe7f385fef8e0efe5ebeb25527d090d0625f60d#diff-578fc2ec77b209d8f0088375d3801054L30-L32

--- a/src/Telegram/Bot/Simple/UpdateParser.hs
+++ b/src/Telegram/Bot/Simple/UpdateParser.hs
@@ -1,5 +1,6 @@
 {-# LANGUAGE DeriveFunctor     #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE CPP #-}
 module Telegram.Bot.Simple.UpdateParser where
 
 import           Control.Applicative
@@ -27,8 +28,10 @@ instance Monad UpdateParser where
   return = pure
   UpdateParser x >>= f = UpdateParser (\u -> x u >>= flip runUpdateParser u . f)
 
+#if MIN_VERSION_base(4,13,0)
 instance MonadFail UpdateParser where
   fail _ = empty
+#endif
 
 mkParser :: (Update -> Maybe a) -> UpdateParser a
 mkParser = UpdateParser

--- a/src/Telegram/Bot/Simple/UpdateParser.hs
+++ b/src/Telegram/Bot/Simple/UpdateParser.hs
@@ -27,6 +27,9 @@ instance Monad UpdateParser where
   return = pure
   UpdateParser x >>= f = UpdateParser (\u -> x u >>= flip runUpdateParser u . f)
 
+instance MonadFail UpdateParser where
+  fail _ = empty
+
 mkParser :: (Update -> Maybe a) -> UpdateParser a
 mkParser = UpdateParser
 

